packages/input-option/src/InputOption/InputOption.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+
+import { cx } from '@leafygreen-ui/emotion';
+import { useDarkMode } from '@leafygreen-ui/leafygreen-provider';
+import {
+  Polymorphic,
+  PolymorphicAs,
+  usePolymorphic,
+} from '@leafygreen-ui/polymorphic';
+
+import {
+  inputOptionActiveStyles,
+  inputOptionDisabledStyles,
+  inputOptionHoverStyles,
+  inputOptionStyles,
+  inputOptionThemeStyles,
+  inputOptionWedge,
+} from './InputOption.style';
+import { InputOptionProps } from './InputOption.types';
+
+export const InputOption = Polymorphic<InputOptionProps>(
```

Comment:
Should this component respond to the "size" prop as well?

Reply:
Not necessarily, since not implementing components will have the same `Size` values (at least not until we have a unified token). It could have a `baseFontSize` prop though